lib/ae_mdw/names.ex

```diff
@@ -326,43 +326,29 @@ defmodule AeMdw.Names do
       transfers: Enum.map(transfers, &expand_txi(Format.bi_txi_txi(&1), expand?)),
       revoke: (revoke && expand_txi(Format.bi_txi_txi(revoke), expand?)) || nil,
       auction_timeout: auction_timeout,
-      pointers: render_pointers(updates),
-      ownership: render_ownership(claims, transfers)
+      pointers: render_pointers(name),
+      ownership: render_ownership(name)
     }
   end
 
-  defp render_pointers([]) do
-    %{}
-  end
-
-  defp render_pointers([{_bi, txi} | _rest_updates]) do
-    txi
-    |> Txs.fetch!()
+  defp render_pointers(name) do
+    name
+    |> Name.pointers()
+    |> Enum.reverse()
+    |> Enum.at(0)
```

Comment:
Good trick. It would be good to have a function to indicate it's getting the biggest key from pointer_kv_raw map with a comment on why the biggest key is the one that matters (from existing code `:aens_pointer.key(ptr)` it is clear if it would be monotonic according to pointer height).

Reply:
Added a comment to it, we should investigate the reason a bit further though